clients/unshaded/src/main/java/tachyon/client/FileSystemMasterClient.java

```diff
@@ -467,4 +467,29 @@ public synchronized boolean unmount(TachyonURI tachyonPath) throws IOException {
     }
     throw new IOException("Failed after " + retry + " retries.");
   }
+
+  /**
+   * Persists the given Tachyon file with the given length
```

Comment:
I thought our new style checker will raise error if this statement doesn't end with "."
surprise to see it passes the check.